test/cli/test/exec-subresource/deny-exec-by-pod-label.yaml

```diff
@@ -20,7 +20,7 @@ spec:
         all:
           - resources:
               kinds:
-                - PodExecOptions
+                - Pod/exec
```

Comment:
cc @Vyom-Yadav

Reply:
https://github.com/kyverno/website/issues/793 to track doc updates.

Reply:
Those kinds are not really well known, very few users know about `PodExecOptions`. Removing it is not a problem, just that policies with `PodExecOptions` will not work in the latest version.